pkgs/development/python-modules/jupyter-ui-poll/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, hatchling
+, ipython
+}:
+
+buildPythonPackage rec {
+  pname = "jupyter-ui-poll";
+  version = "0.2.2";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchFromGitHub {
+    owner = "Kirill888";
+    repo = "jupyter-ui-poll";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-DWZFvzx0aNTmf1x8Rq19OT0PFRxdpKefWYFh8C116Fw";
+  };
+
```

According to [`setup.cfg`](https://github.com/Kirill888/jupyter-ui-poll/blob/v0.2.2/setup.cfg#L35-L36). Interesting that didn't lead to a failure.

```suggestion
  nativeBuildInputs = [ setuptools ];
```